### DISCUSSION TITLE
[Feature] Add option to set separate directory for thumbnails

### DISCUSSION THREAD

**Discussion by micash545** [immich-app/immich#9683] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add option to set completely different directory for thumbnails etc.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by schuhbacca** [comment]
https://immich.app/docs/guides/custom-locations/
